app/recordtransfer/emails.py

```diff
@@ -69,20 +81,31 @@ def send_submission_creation_success(form_data: dict, submission: Submission) ->
 
 
 @django_rq.job
-def send_submission_creation_failure(form_data: dict, user_submitted: User) -> None:
+def send_submission_creation_failure(
+    form_data: dict, user_submitted: User, recipient_emails: Optional[List[str]] = None
+) -> None:
     """Send an email to users who get submission email updates that a user submitted a new
     submission and there WERE errors.
 
     Args:
         form_data (dict): A dictionary of the cleaned form data from the submission form. This is
             NOT the CAAIS tree version of the form.
         user_submitted (User): The user that tried to create the submission.
+        recipient_emails (List[str], optional): A list of recipient email addresses to send the notification to. If not provided, admin recipients will be used.
```

Comment:
I'm seeing this linting error here:

> Doc line too long (160 > 99) Ruff [W505](https://docs.astral.sh/ruff/rules/doc-line-too-long)

Reply:
We'll add this file to the linting action soon, yes. For now it's OK that there are errors in this file, my plan is still to very slowly expand the scope of the linting